toolchain/check/import_ref.cpp

```diff
@@ -413,6 +413,29 @@ class ImportRefResolver {
     return context_.inst_blocks().AddCanonical(contents);
   }
 
+  auto GetLocalGenericInstanceArgs(SemIR::GenericInstanceId instance_id)
```

Comment:
Might be good to have comments on these new functions, explaining the intent.

Reply:
Done.